src/visitor/FindAnyExprVisitor.h

```diff
@@ -17,7 +17,7 @@ namespace graph {
 
 class FindAnyExprVisitor final : public ExprVisitorImpl {
 public:
-    explicit FindAnyExprVisitor(const std::unordered_set<Expression::Kind>& kinds);
+    explicit FindAnyExprVisitor(const std::unordered_set<Expression::Kind> kinds);
```

Comment:
Why update this?

Reply:
I want use temporary variable:
```
            graph::FindAnyExprVisitor visitor(
                {Expression::Kind::kInputProperty, Expression::Kind::kVarProperty});
```

Reply:
Because I want pass a temorary variable to the visitor before:
```
FindAnyExprVisitor visitor({Expression::kind::kInput, Expression::kind::kVarProperty}).
```
But this will introduce an additional copy so I changed it back.